.claude/agents/test-validator.md

```
@@ -0,0 +1,76 @@
+
+실패하는 테스트의 **원인을 진단만** 한다. **코드를 절대 수정하지 않는다.**
+
+## 절차
+
+1. 테스트를 실행해 실패를 재현한다.
+
+```bash
+./gradlew test --tests "{클래스명}" --info
+```
+
+2. 실패 메시지와 스택 트레이스를 확보한다. 리포트: `build/reports/tests/test/index.html`
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)